NetKAN/AntennaRangePatches.netkan

```diff
@@ -0,0 +1,30 @@
+{
+	"spec_version"	: "v1.4",
+	"identifier"	: "AntennaRangePatches",
+	"$kref"		: "#/ckan/kerbalstuff/1220",
+	"$vref"		: "#/ckan/ksp-avc",
+	"license": "CC-BY-NC-SA-4.0",
+	"resources"	:
+	{
+		"kerbalstuff" : "https://kerbalstuff.com/mod/1220",

```

Comment:
On 10/28/15 9:17 AM, Olympic1 wrote:
>
> In NetKAN/AntennaRangePatches.netkan 
> <https://github.com/KSP-CKAN/NetKAN/pull/2494#discussion_r43252857>:
>
> > @@ -0,0 +1,30 @@
> > +{
> > +	"spec_version"	: "v1.4",
> > +	"identifier"	: "AntennaRangePatches",
> > +	"$kref"		: "#/ckan/kerbalstuff/1220",
> > +	"$vref"		: "#/ckan/ksp-avc",
> > +	"license": "CC-BY-NC-SA-4.0",
> > +	"resources"	:
> > +	{
> > +		"kerbalstuff" : "https://kerbalstuff.com/mod/1220",
>
> You can remove this line
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/KSP-CKAN/NetKAN/pull/2494/files#r43252857>.
>

Can I assume that no action is needed by me anymore on this?

If not, then which line can be removed, the kerbalstuff line?


thanks


JBB